engine/src/main/java/org/hibernate/validator/internal/engine/ConstraintViolationImpl.java

```diff
@@ -39,7 +39,6 @@
 	private final Map<String, Object> messageParameters;
 	private final Map<String, Object> expressionVariables;
 	private final Class<T> rootBeanClass;
-	private final ElementType elementType;
```

Comment:
I've remembered what I wanted to ask about this commit :) I was just curious why we had `ElementType` in here in the first place ? Everything else in here looks fine.

Reply:
No idea tbh. Probably a leftover of some sort.